### ISSUE TITLE
Can the current version cancel all the open orders?

### ISSUE THREAD

**Issue by BIGANDWET** [Jamonek/Robinhood#37]
If not is there a way a get open orders id and cancel it.

I'm very new to programming. I appreciate your responses.

**Comment by Jamonek** [comment]
I'll look into this

**Comment by josh-ua** [comment]
@Jamonek @BIGANDWET I'm working on this along with singular order cancelling functionality!

**Comment by Jamonek** [comment]
@josh-ua looking forward to see what you come up with!

**Comment by bdowling** [comment]
fwiw, I started creating this myself as I want to start using this API in my algo trading.

I saw this issue, but no updates and did not realize the PR #40 was already in the works as it was not linked to this issue.

fwiw, see my diff here https://github.com/Jamonek/Robinhood/compare/master...bdowling:add_cancel_order

I think there may be some valuable points in both our implementations.  I should add the raiseError on status as well, I may do that and submit a PR if you guys would like.
